scripts/apps/authoring/styles/authoring.scss

```diff
@@ -212,99 +212,150 @@
 // Opened articles
 // -------------------------------------------------
 
-.opened-articles {
+$bottom-bar-item-border-color: rgba(255,255,255,0.1);
+$bottom-bar-item-border-color-active: transparent;
+
+.opened-articles-bar {
     position: fixed;
-    min-height: $authoring-opened-articles;
-    left: 0; right: 0;
+    height: $authoring-opened-articles;
+    left: 0; 
+    right: 0;
     bottom: 0;
-    background: rgba(0,0,0,0.85);
+    background: rgba(20, 20, 20, 0.95);
     z-index: $zindexFixedNavbar;
-
-    .opened-articles__list > li {
-        @include border-box();
-        padding: 0 1.6rem;
-        max-width: 24rem;
+    white-space: nowrap;
+    max-width: 100vw;
+    display: grid;
+    grid-template-rows: 32px;
+    grid-template-columns: 48px 1fr auto auto;
```

Comment:
if that `48px` is for aligning the button with the sidebar, can you make the two to use the same variable for the width?

Reply:
yep, this should be done.